packages/ui/components/typography/index.styl

```diff
@@ -1,2 +1,32 @@
 .text
   font(xxl)
+
+.h1
```

Comment:
_scales = h1 h2 h3 h4 h5 h6 body1 body2

for scale in _scales
  .{scale}
    font(scale)